src/test/groovy/org/prebid/server/functional/service/PrebidServerService.groovy

```diff
@@ -141,6 +141,14 @@ class PrebidServerService implements ObjectMapperWrapper {
         response.as(CookieSyncResponse)
     }
 
+    @Step("[POST] /cookie_sync with uids cookie")
+    CookieSyncResponse sendCookieSyncRequest(CookieSyncRequest request, String uidsCookie) {
```

Comment:
What was the point of duplicating the method, but using a String instead of POJO?